### DISCUSSION TITLE
How to create multiple blog pages?

### DISCUSSION THREAD

**Discussion by ghost** [MrGreensWorkshop/MrGreen-JekyllTheme#107] [Problems] [answered]
I have no experience with Ruby, Jekyll nor liquid template engine so I would like to ask if there's an easy way to create another blog page with his own navigation button. I need to duplicate things, that's for sure, but I would like to ask for straightforward guidance.

Thanks in advance!

**Comment by MrGreensWorkshop** [comment, marked as answer]
Himm, I don't think that would be easy. Blog post pages involves liquid, and javascripts. I think that would be like redesigning everything from start.
